crates/fluvio-hub-util/src/fvm/api.rs

```diff
@@ -15,9 +13,8 @@ pub struct Client {
 
 impl Client {
     /// Creates a new [`Client`] with the default Hub API URL
-    #[allow(unused)]
-    pub fn new() -> Result<Self> {
-        let api_url = Url::parse(HUB_REMOTE)?;
+    pub fn new(url: &str) -> Result<Self> {
+        let api_url = url.parse::<Url>()?;
 
         Ok(Self { api_url })
     }
```

Comment:
This allows us to set different URLs

Reply:
This can be passed as env var rather than hard coding.  So this means
1. Pass base URL.
2. Set base URL in the CLI (this will automatically read env var)

Reply:
Agree! Im now removing the hardcoded `HUB_REMOTE` constant and allowing the user to
pass the `url` to use.

As can be seen in the demo I just posted above, the `--registry` option can be passed
to determine the URL to use when fetching the Package Set from the Hub.

Additionally I can add an environment variable: `HUB_REGISTRY_URL` as a second approach
to pass the Hub URL, perhaps this is helpful for CI as well!